### ISSUE TITLE
Smooth scrolling makes M-v/Page Up scrolling down

### ISSUE THREAD

**Issue by qqldd** [syl20bnr/spacemacs#6379]
#### Description
M-v/ Page Up scrolls down a few lines. It should scroll up a screen

#### Reproduction guide
- Start Emacs
- Open .spacemacs file
- M-g g 100
- M-v or Page Up

*Observed behaviour:*
- The cursor now is at line 109

*Expected behaviour:*
- The cursor should before line 100

#### System Info
- OS: gnu/linux
- Emacs: 24.5.1
- Spacemacs: 0.105.21
- Spacemacs branch: master (rev. 0283f64)
- Graphic display: t
- Distribution: spacemacs
- Editing style: emacs
- Completion: helm
- Layers:
```elisp
(emacs-lisp c-c++ shell-scripts python csv shell sql markdown latex javascript html git perforce auto-completion gtags irony-mode semantic colors themes-megapack sr-speedbar org eyebrowse ibuffer)
```

#### Backtrace
```
BACKTRACE IF RELEVANT
```

#### Additions:
- It also happens in *.py and *.cpp file. If I disable smooth-scrolling, it seems like the problem goes.

**Comment by TheBB** [comment]
This is fixed in develop already.

**Comment by TheBB** [comment]
https://github.com/syl20bnr/spacemacs/issues/6097 https://github.com/syl20bnr/spacemacs/issues/1103 https://github.com/syl20bnr/spacemacs/issues/6316

**Comment by d12frosted** [comment]
Fixed with released of Spacemacs v0.200. Let us know if you still have any problems with this issue.
